tools/gyp/pylib/gyp/generator/msvs.py

```diff
@@ -3416,7 +3416,7 @@ def _FinalizeMSBuildSettings(spec, configuration):
     )
     # Turn on precompiled headers if appropriate.
     if precompiled_header:
-        precompiled_header = os.path.split(precompiled_header)[1]
+        # precompiled_header = os.path.split(precompiled_header)[1]
```

Comment:
We also need to either make this conditional or find out how to make precompiled headers work with ClangCL

Reply:
I skipped precompiled headers when `clang==1` in the new PR.